packages/blade/src/components/Icons/MailIcon/MailIcon.native.test.tsx

```diff
@@ -0,0 +1,11 @@
+import EmailIcon from '.';
```

Comment:
Let's use the same terminology:

```suggestion
import MailIcon from '.';
```